system/composite/composite_main.c

```diff
@@ -55,6 +55,12 @@
 #  endif
 #endif
 
+#ifdef CONFIG_FILE_STREAM
+#  define printerr(fmt, ...) fprintf(stderr, (fmt), ##__VA_ARGS__)
+#else
+#  define printerr(fmt, ...) dprintf(STDERR_FILENO, (fmt), ##__VA_ARGS__)
```

Comment:
why not always use dprintf?

Reply:
Good question, I did not write the composite_main.c code.

The problem for me is I'm running out of flash memory

Reply:
I think it' more simple to replace fprintf with dprintf.

Reply:
Ok, I can convert this file because this is a blocker issue for me (the SW does not build any more)

Reply:
Sure, let's fix it one by one.